src/lifetimes/borrow-both.md

```diff
@@ -0,0 +1,46 @@
+---
+minutes: 5
+---
+
+# Borrow Both
+
+In this case, we have a function where either `a` or `b` may be returned. In
+this case we use the lifetime annotations to tell the compiler that both borrows
+may flow into the return value.
+
+```rust
+fn pick<'a>(c: bool, a: &'a i32, b: &'a i32) -> &'a i32 {
+    if c { a } else { b }
+}
+
+fn main() {
+    let mut a = 5;
+    let mut b = 10;
+
+    let r = pick(true, &a, &b);
+
+    // Which one is borrowed? Should either of these
+    // be allowed?
```

Comment:
Please update the comment here to match the other slide if you accept the suggested edit there.